Automattic/wp-calypso
# Partner Portal: Add static license listing UI

Context:
* Project thread: pbtFPC-Um-p2
* i4 designs: pbtFPC-103-p2

#### Changes proposed in this Pull Request

* Introduce the license listing UI with hardcoded placeholder data.

#### Testing instructions

* If you do not have a partner key, please contact Infinity for one or you will be unable to view the UI.
* Checkout PR locally, run `yarn && yarn start-jetpack-cloud`.
* Visit http://jetpack.cloud.localhost:3000/partner-portal and select your partner key, if prompted.
* Confirm the license list UI you are presented with matches designs (minus search, sort, filter, and revoke as they will be added in a separate PR).

Note: Mobile styles were done ad-hoc.
Note: The Partner Portal is only enabled in the development environment for now.

![license-listing](https://user-images.githubusercontent.com/22746396/106746313-0943d780-662b-11eb-8583-e48defd9d095.png)
